pkg/deployer/lib/controller.go

```diff
@@ -285,22 +279,22 @@ func (c *controller) reconcilePrivate(ctx context.Context, metadata *metav1.Part
 	// Deployitem has been initialized, proceed with reconcile/delete
 
 	if di.DeletionTimestamp.IsZero() {
-		lsError := c.reconcile(ctx, lsCtx, di, rt)
-		err := c.handleReconcileResult(ctx, lsError, old, di)
-		return c.buildResult(di.Status.Phase, err)
+		lsError := c.reconcile(ctx, di, rt)
+		_ = c.handleReconcileResult(ctx, lsError, old, di)
```

Comment:
I don't get this change. This function returns an error, if updating the status of the DeployItem failed. We want this error to be returned, so that the controller-runtime requeues the DeployItem so that it gets the correct status, don't we? Why would we ignore this error?

Reply:
In the next line `return c.buildResult(di.Status.Phase)` the re-queue logic is implemented. It only depends on the status of the deploy item and not the error. It is long ago when Robert changed this. As far as I remember we wanted to prevent timeouts due to exponential back-offs. In this change I have just cleaned up a  not used parameter in the method `c.buildResult(...)`